src/js/exercisePopUp.js

```diff
@@ -0,0 +1,110 @@
+import { serviceGetExercisesById } from './services.js';
+
+const exercisePopUpBackdrop = document.querySelector('.exercise-pop-up-backdrop');
+const exercisePopUpContent = document.querySelector('.exercise-pop-up-content');
+const exercisePopUpCloseBtn = document.querySelector('.exercise-pop-up-close-btn');
```

Comment:
please move to constants.js, refs object

Reply:
Done